alphabase/psm_reader/sage_reader.py

```diff
@@ -562,7 +563,7 @@ def _sage_spec_idx_from_scan_nr(scan_indicator_str: str) -> int:
     return int(re.search(r"scan=(\d+)", scan_indicator_str).group(1)) - 1
 
 
-class SageReaderBase(PSMReaderBase):
+class SageReaderBase(PSMReaderBase, ABC):
```

Comment:
why do we need to inherit the ABC here?

Reply:
bc `SageReaderBase` should not be instantiated